doc/how_to_customize.rst

```diff
@@ -0,0 +1,54 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _how_to_customize:
+
+##########################################
+How to customize?
```

Comment:
I'd say: how to use PyPSA-Earth to your energy problem? Or something like that.
Customization may be interpreted how to modify also the source for example. Or maybe this is what I may also understand (I may be wrong though)

Reply:
Agree, that is not obvious that data adjustment is meant. Tried to improve.